e2e/nomostest/oci-image-verification-server.go

```diff
@@ -141,9 +141,11 @@ func auth(nt *NT) error {
 	sharedTmpDir := filepath.Join(os.TempDir(), NomosE2E, nt.ClusterName)
 	sharedTestRegistrySSLDir := filepath.Join(sharedTmpDir, string(RegistrySyncSource), sslDirName)
 	testRegistryCACertPath := filepath.Join(sharedTestRegistrySSLDir, caCertFile)
-	if err := createSecret(nt, OCISignatureVerificationNamespace, PublicCertSecretName(RegistrySyncSource),
-		fmt.Sprintf("cert=%s", testRegistryCACertPath)); err != nil {
-		return err
+	if e2e.ArtifactRegistry == "local" {
```

Comment:
`e2e.ArtifactRegistry` is a constant. It will never be `local`.

I think you meant ` if *e2e.OCIProvider == e2e.Local`?

Reply:
You can also move the lines above inside the if condition. They are not used when using GAR.

Reply:
The comment also needs to be updated.

Reply:
Done